### PR TITLE
style: add avatar & badge components

### PR DESCRIPTION
**Pull request by anik31**
Inside components folder avatar.css & badge.css contains the css for the avatar & badge components respectively.

docs.html contains the html for all the components.

Deploy preview - https://deploy-preview-1--vacay-ui.netlify.app/docs

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -31,7 +31,7 @@ html{
     font-family: 'Roboto', sans-serif;
     font-size: 100%;
     color: var(--black-color);
-    background-color: var(--lynx-white-color);
+    background-color: var(--white-color);
 }
 
 button{
```
Comment (vrx29) [inline style.css:37]: empty declaration can be removed

**Review by partha8 — APPROVED** [review]
LGTM

File: style.css (hunk 1 above)
Comment (anik31) [reply style.css:37]: can you please explain ? I didn't understand.

**Review by hsnice16 — CHANGES_REQUESTED** [review]
Very clean UI man. Nice work on it.

Just one suggestion, to increase readability you can make variable name capitalized like `--PRIMARY-COLOR` or `--COLOR-PRIMARY`. whatever suits you.

**Comment by anik31** [comment]
> Very clean UI man. Nice work on it.
> 
> Just one suggestion, to increase readability you can make variable name capitalized like `--PRIMARY-COLOR` or `--COLOR-PRIMARY`. whatever suits you.

Thanks for the suggestion @hsnice16 but I prefer lowercase as mentioned in MDN & also it remains consistent.

**Comment by anik31** [comment]
> LGTM

Ty for reviewing @partha8

**Review by vrx29 — APPROVED** [review]
looks good you can go ahead.

**Comment by anik31** [comment]
> looks good you can go ahead.

Thanks for the review.

**Pull request merged**